### PR TITLE
Dcs availability zones

### LINKED ISSUE #148: [DCS] - Implementation of some API's missing
There's currently no implementation for querying maintenance time windows, service specifications or AZ informations.

See here:
https://docs.otc.t-systems.com/api/dcs/dcs-api-0312041.html
https://docs.otc.t-systems.com/api/dcs/dcs-api-0312040.html
https://docs.otc.t-systems.com/api/dcs/dcs-api-0312039.html

### PR DESCRIPTION
**Pull request by irina-pereiaslavskaia**
New DCS AZ resource with tests and docs

### CODE REVIEW

File: otcextensions/sdk/dcs/v1/_proxy.py [hunk 1]
```diff
@@ -306,3 +307,22 @@ def service_specifications(self):
...
+
+    # ========= Available Zone ========
+    def availability_zones(self):
+        """Return a generator of Availability Zones where a DCS instance
+        resides.
+
+        :returns: A generator of Availability Zone objects
+        :rtype:
+            :class:
+                `~otcextensions.sdk.dcs.v1.availability_zone.AvailabilityZone`
```
Comment (vladimirvshivkov) [inline otcextensions/sdk/dcs/v1/_proxy.py:319]: I'm wondering how you managed to fit the line.

**Review by vladimirvshivkov — APPROVED** [review]

**Review by SebastianGode — APPROVED** [review]

File: otcextensions/sdk/dcs/v1/availability_zone.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from openstack import resource
+
+
+class AvailabilityZone(resource.Resource):
+
+    resources_key = 'available_zones'
```
Comment (anton-sidelnikov) [inline otcextensions/sdk/dcs/v1/availability_zone.py:18]: i think better to keep common name availability_zones

Comment (anton-sidelnikov) [reply otcextensions/sdk/dcs/v1/availability_zone.py:18]: check if you can change it

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: otcextensions/sdk/dcs/v1/availability_zone.py (hunk 2 above)
Comment (irina-pereiaslavskaia) [reply otcextensions/sdk/dcs/v1/availability_zone.py:18]: 'available_zones' returns in content of response that's way I use it

**Review by anton-sidelnikov — APPROVED** [review]

**Comment by lego963** [comment]
regate

**Pull request merged**